docs/cmake/index.md

```diff
@@ -0,0 +1,16 @@
+# CMake configuration guide
+
+:::{toctree}
```

Comment:
You can use triple backticks here instead of `:::` to play nicely with `mdformat`